src/main/kotlin/com/hibob/kotlinBasic/controlStructures.kt

```diff
@@ -0,0 +1,31 @@
+package com.hibob.kotlinBasic
+
+fun isValidIdentifier(s: String): Boolean {
+
+    if (s.isEmpty()) return false
+    if (s[0] == '_') return true
```

Comment:
No... If it starts with an underscore it doesn't mean that the whole identifier is valid. You still need to check that all characters are permitted!